<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Imaging tools | Informatics Technology for Cancer Research (ITCR): Cancer Informatics for PIs</title>
  <meta name="description" content="This book provides an introduction to tools and key responsibilities for lab leaders conducting informatics for cancer research." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Imaging tools | Informatics Technology for Cancer Research (ITCR): Cancer Informatics for PIs" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This book provides an introduction to tools and key responsibilities for lab leaders conducting informatics for cancer research." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Imaging tools | Informatics Technology for Cancer Research (ITCR): Cancer Informatics for PIs" />
  
  <meta name="twitter:description" content="This book provides an introduction to tools and key responsibilities for lab leaders conducting informatics for cancer research." />
  

<meta name="author" content="Carrie Wright" />


<meta name="date" content="2021-03-23" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="itcr-tools.html"/>
<link rel="next" href="computing-for-cancer-research.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>



</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this Course</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#motivation"><i class="fa fa-check"></i><b>1.1</b> Motivation</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html"><i class="fa fa-check"></i><b>2</b> Cancer Informatics for Research Leaders</a><ul>
<li class="chapter" data-level="2.1" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#guidelines-for-working-with-informatics-experts"><i class="fa fa-check"></i><b>2.1</b> Guidelines for working with informatics experts</a><ul>
<li class="chapter" data-level="2.1.1" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#finding-and-creating-good-relationships-for-informatics-projects"><i class="fa fa-check"></i><b>2.1.1</b> Finding and creating good relationships for informatics projects</a></li>
<li class="chapter" data-level="2.1.2" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#identifying-good-informatics-questions"><i class="fa fa-check"></i><b>2.1.2</b> Identifying good informatics questions</a></li>
<li class="chapter" data-level="2.1.3" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#starting-a-project"><i class="fa fa-check"></i><b>2.1.3</b> Starting a project</a></li>
<li class="chapter" data-level="2.1.4" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#good-informatics-leader-practices"><i class="fa fa-check"></i><b>2.1.4</b> Good informatics leader practices</a></li>
<li class="chapter" data-level="2.1.5" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#informatics-project-pitfalls"><i class="fa fa-check"></i><b>2.1.5</b> Informatics project pitfalls</a></li>
<li class="chapter" data-level="2.1.6" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#informatics-lab-management-tools"><i class="fa fa-check"></i><b>2.1.6</b> Informatics lab management tools</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#collaborating-with-informatics-experts"><i class="fa fa-check"></i><b>2.2</b> Collaborating with Informatics Experts</a><ul>
<li class="chapter" data-level="2.2.1" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#the-benefits-of-collaboration"><i class="fa fa-check"></i><b>2.2.1</b> The benefits of collaboration</a></li>
<li class="chapter" data-level="2.2.2" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#potential-challenges"><i class="fa fa-check"></i><b>2.2.2</b> Potential Challenges</a></li>
<li class="chapter" data-level="2.2.3" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#references"><i class="fa fa-check"></i><b>2.2.3</b> References</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#employing-informatics-experts"><i class="fa fa-check"></i><b>2.3</b> Employing informatics experts</a></li>
<li class="chapter" data-level="2.4" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#mentoring-informatics-students"><i class="fa fa-check"></i><b>2.4</b> Mentoring informatics students</a><ul>
<li class="chapter" data-level="2.4.1" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#co-mentoringcollaborativeteam-mentoring"><i class="fa fa-check"></i><b>2.4.1</b> Co-mentoring/Collaborative/team mentoring</a></li>
<li class="chapter" data-level="2.4.2" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#peer-mentoring"><i class="fa fa-check"></i><b>2.4.2</b> Peer mentoring</a></li>
<li class="chapter" data-level="2.4.3" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#electronic-mentoring"><i class="fa fa-check"></i><b>2.4.3</b> Electronic mentoring</a></li>
<li class="chapter" data-level="2.4.4" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#career-goals"><i class="fa fa-check"></i><b>2.4.4</b> Career goals</a></li>
<li class="chapter" data-level="2.4.5" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#mentoring-underrepresented-students"><i class="fa fa-check"></i><b>2.4.5</b> Mentoring underrepresented students</a></li>
<li class="chapter" data-level="2.4.6" data-path="cancer-informatics-for-research-leaders.html"><a href="cancer-informatics-for-research-leaders.html#refrences"><i class="fa fa-check"></i><b>2.4.6</b> Refrences</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="informatics-tools-for-cancer-research.html"><a href="informatics-tools-for-cancer-research.html"><i class="fa fa-check"></i><b>3</b> Informatics tools for Cancer Research</a></li>
<li class="chapter" data-level="4" data-path="itcr-tools.html"><a href="itcr-tools.html"><i class="fa fa-check"></i><b>4</b> ITCR Tools</a></li>
<li class="chapter" data-level="5" data-path="imaging-tools.html"><a href="imaging-tools.html"><i class="fa fa-check"></i><b>5</b> Imaging tools</a><ul>
<li class="chapter" data-level="5.1" data-path="imaging-tools.html"><a href="imaging-tools.html#tools-for-slide-image-analysis"><i class="fa fa-check"></i><b>5.1</b> Tools for slide image analysis</a></li>
<li class="chapter" data-level="5.2" data-path="imaging-tools.html"><a href="imaging-tools.html#tools-for-radiology-image-analysis"><i class="fa fa-check"></i><b>5.2</b> Tools for Radiology Image Analysis</a><ul>
<li class="chapter" data-level="5.2.1" data-path="imaging-tools.html"><a href="imaging-tools.html#dslicer"><i class="fa fa-check"></i><b>5.2.1</b> <span>3Dslicer</span></a></li>
<li class="chapter" data-level="5.2.2" data-path="imaging-tools.html"><a href="imaging-tools.html#tools-for-medical-image-standardization"><i class="fa fa-check"></i><b>5.2.2</b> Tools for Medical Image Standardization</a></li>
<li class="chapter" data-level="5.2.3" data-path="imaging-tools.html"><a href="imaging-tools.html#imaging-archivesrepositories"><i class="fa fa-check"></i><b>5.2.3</b> Imaging Archives/Repositories:</a></li>
<li class="chapter" data-level="5.2.4" data-path="imaging-tools.html"><a href="imaging-tools.html#specialized-tools"><i class="fa fa-check"></i><b>5.2.4</b> Specialized tools:</a></li>
<li class="chapter" data-level="5.2.5" data-path="imaging-tools.html"><a href="imaging-tools.html#galaxy"><i class="fa fa-check"></i><b>5.2.5</b> Galaxy</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html"><i class="fa fa-check"></i><b>6</b> Computing for Cancer Research</a><ul>
<li class="chapter" data-level="6.1" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#computing"><i class="fa fa-check"></i><b>6.1</b> Computing</a><ul>
<li class="chapter" data-level="6.1.1" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#core-computing-components"><i class="fa fa-check"></i><b>6.1.1</b> Core Computing Components</a></li>
<li class="chapter" data-level="6.1.2" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#current-computer-capacity"><i class="fa fa-check"></i><b>6.1.2</b> Current Computer Capacity</a></li>
<li class="chapter" data-level="6.1.3" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#servers"><i class="fa fa-check"></i><b>6.1.3</b> Servers</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#computing-platforms"><i class="fa fa-check"></i><b>6.2</b> Computing Platforms</a></li>
<li class="chapter" data-level="6.3" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#hardware"><i class="fa fa-check"></i><b>6.3</b> Hardware</a><ul>
<li class="chapter" data-level="6.3.1" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#personal-computers"><i class="fa fa-check"></i><b>6.3.1</b> <strong>Personal computers</strong></a></li>
<li class="chapter" data-level="6.3.2" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#shared-computers"><i class="fa fa-check"></i><b>6.3.2</b> <strong>Shared Computers</strong></a></li>
<li class="chapter" data-level="6.3.3" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#accessing-shared-computer-resources"><i class="fa fa-check"></i><b>6.3.3</b> Accessing Shared Computer Resources</a></li>
<li class="chapter" data-level="6.3.4" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#cloud-computing"><i class="fa fa-check"></i><b>6.3.4</b> <strong>Cloud Computing</strong></a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#choosing-a-computing-platform"><i class="fa fa-check"></i><b>6.4</b> Choosing a Computing Platform</a><ul>
<li class="chapter" data-level="6.4.1" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#important-questions"><i class="fa fa-check"></i><b>6.4.1</b> Important questions</a></li>
<li class="chapter" data-level="6.4.2" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#benefits-of-cloud-computing"><i class="fa fa-check"></i><b>6.4.2</b> Benefits of Cloud Computing</a></li>
<li class="chapter" data-level="6.4.3" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#challenges-of-cloud-computing"><i class="fa fa-check"></i><b>6.4.3</b> Challenges of Cloud Computing</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#local-costs"><i class="fa fa-check"></i><b>6.5</b> Local Costs</a></li>
<li class="chapter" data-level="6.6" data-path="computing-for-cancer-research.html"><a href="computing-for-cancer-research.html#cloud-costs"><i class="fa fa-check"></i><b>6.6</b> Cloud Costs</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="data-management-for-cancer-research.html"><a href="data-management-for-cancer-research.html"><i class="fa fa-check"></i><b>7</b> Data Management for Cancer Research</a><ul>
<li class="chapter" data-level="7.1" data-path="data-management-for-cancer-research.html"><a href="data-management-for-cancer-research.html#good-practice"><i class="fa fa-check"></i><b>7.1</b> Good Practice</a></li>
<li class="chapter" data-level="7.2" data-path="data-management-for-cancer-research.html"><a href="data-management-for-cancer-research.html#data-security"><i class="fa fa-check"></i><b>7.2</b> Data Security</a></li>
<li class="chapter" data-level="7.3" data-path="data-management-for-cancer-research.html"><a href="data-management-for-cancer-research.html#data-privacy"><i class="fa fa-check"></i><b>7.3</b> Data Privacy</a></li>
<li class="chapter" data-level="7.4" data-path="data-management-for-cancer-research.html"><a href="data-management-for-cancer-research.html#data-reproducibility"><i class="fa fa-check"></i><b>7.4</b> Data REproducibility</a><ul>
<li class="chapter" data-level="7.4.1" data-path="data-management-for-cancer-research.html"><a href="data-management-for-cancer-research.html#nih-data-sharing-policies"><i class="fa fa-check"></i><b>7.4.1</b> NIH Data Sharing Policies</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Informatics Technology for Cancer Research (ITCR): Cancer Informatics for PIs</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="imaging-tools" class="section level1">
<h1><span class="header-section-number">Chapter 5</span> Imaging tools</h1>
<div id="tools-for-slide-image-analysis" class="section level2">
<h2><span class="header-section-number">5.1</span> Tools for slide image analysis</h2>
<p><a href="https://www.csb.pitt.edu/ith/">Thrive</a> is a very comprehensive interactive GUI-based tool that may be better for less computationally experienced users and for users who wish to try a variety of algorithms for image processing using different tumor heterogeneity/diversity, cell segmentation, and cell quantification.</p>
<p><img src="images/thrive1.png" /></p>
<p>[<a href="https://www.csb.pitt.edu/ith/">Source</a>]</p>
<p><img src="images/thrive2.png" /></p>
<p>[<a href="https://www.csb.pitt.edu/ith/">Source</a>]</p>
<p>Another tool being developed is called the <a href="http://pathiip.org/">Pathology Image Informatics Platform (PIIP)</a> that will be a similar all-in-one platform for viewing, annotating, sharing and managing whole slide images. There are plans to further develop the platform to interface with third party algorithms for segmentation, color normalization, biomarker quantification, and radiology-pathology fusion. This platform will use the Pathcore <a href="https://pathcore.com/sedeen/">Sedeen slide viewer</a>.</p>
<p>For more specific tools check out:</p>
<p>The Pathcore <a href="https://pathcore.com/sedeen/">Sedeen slide viewer</a>, which allows for color correction, pseudo-coloring, cropping, resizing, overlays of additional layers, registration, markup, and annotation. It currently supports Aperio SVS, Leica SCN, other TIFF-based formats, and JPEG-2000 images. Users can even create their own tools.</p>
<p><a href="https://youtu.be/K34Gqt1ItYs" class="uri">https://youtu.be/K34Gqt1ItYs</a></p>
<p>The <a href="https://sbu-bmi.github.io/quip_distro/">Quantitative Imaging in Pathology (QUIP)</a> is a web-based tool that allows for interactive analysis, exploration, and management of whole slide tissue images. It also allows for users to create plots based on image features using FeatureScape. (requires docker and git)</p>
<p><a href="https://youtu.be/dK4c6ti1Dvc" class="uri">https://youtu.be/dK4c6ti1Dvc</a></p>
<p><a href="https://github.com/choosehappy/HistoQC">HistoQC</a> - is an open-source tool written in python for examining slides for artifacts and computing slide characteristics (stain intensity etc.) for quality control and data standardization that uses <a href="https://openslide.org/demo/">OpenSlide</a> for viewing slides.</p>
</div>
<div id="tools-for-radiology-image-analysis" class="section level2">
<h2><span class="header-section-number">5.2</span> Tools for Radiology Image Analysis</h2>
<p>There are a variety of tools available to assist with analyzing computed tomography (CT) scans, Magnetic Resonance Imaging (MRI), Positron Emission tomogoraphy (PET) and other nuclear medicine scans, as well as Ultra Sound (US).</p>
<div id="dslicer" class="section level3">
<h3><span class="header-section-number">5.2.1</span> <a href="https://www.slicer.org/">3Dslicer</a></h3>
<p>3D slicer is a powerful tool for CT, MRI, and US analysis, as well as microscopy analysis. This tool allows for interactive 3D visualization (volume rendering), and image processing including registration and segmentation.</p>
<p><img src="images/3DSlicer_intro.png" /></p>
<p>[<a href="https://slicer.readthedocs.io/en/latest/user_guide/getting_started.html">source</a>]</p>
<p>There are many additional 3D slicer extension tools such as:</p>
<ul>
<li><a href="https://github.com/IOWA-PETCT-COSEG/PETCT_Slicer_Extension">PET-CT-CoSeg</a> is a PET-CT co-segmentation extension of 3D Slicer.</li>
<li><a href="http://qin.iibi.uiowa.edu/">many additional PET 3D slicer extensions</a> for normalization, segmentation, extraction of quantitative indices, and automating liver uptake measurements.</li>
<li><a href="http://dmri.slicer.org/">SlicerDMRI</a> performs additional diffusion MRI iamge analyses including command line batch processing, computing scalar maps, and registering multimodal data. It supports both DICOM and nnrd and nhdr images.</li>
</ul>
</div>
<div id="tools-for-medical-image-standardization" class="section level3">
<h3><span class="header-section-number">5.2.2</span> Tools for Medical Image Standardization</h3>
<p><a href="https://qiicr.gitbook.io/dcmqi-guide/">dcmqi (DICOM - digital imaging and communications in medicine (dcm) for Quantitative Imaging (qi))</a> This platform provides libraries and command line tools for standardizing communication of quantitative image analysis using the DICOM standard. This is required for sharing data in some archives like TCIA. See <a href="https://doi.org/10.7717/peerj.2057">this article</a> and <a href="https://doi.org/10.1177/0962280214537333">this article</a> for more information.</p>
<p><a href="https://github.com/OHIF/Viewers">Open Health Imaging Foundation (OHIF) medical imaging viewer</a> (replacing lesion tracker) - web application for viewing, annotating, and reporting on DICOM images in 2D and 3D.</p>
<p><a href="https://epad.stanford.edu/">ePAD</a> A radiology and tissue imaging analysis platform that supports analysis of tumor burden over time. See <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998692/">here</a> for more information.</p>
</div>
<div id="imaging-archivesrepositories" class="section level3">
<h3><span class="header-section-number">5.2.3</span> Imaging Archives/Repositories:</h3>
<ol start="12" style="list-style-type: decimal">
<li>TCIA - radiology and pathology images with annotations and clinical information and in some cases paired omics data
tools here too?</li>
<li>CDSA - pathology data from the Cancer Genome Atlas</li>
</ol>
</div>
<div id="specialized-tools" class="section level3">
<h3><span class="header-section-number">5.2.4</span> Specialized tools:</h3>
<ol start="14" style="list-style-type: decimal">
<li>Breast Radiodensity assessment</li>
<li>TOPAS - ionizing radiation imaging and therapy simulations</li>
<li><a href="https://glioblastoma.alleninstitute.org/ish/">Ivy Glioblastoma Atlas Project (Ivy GAP)</a> data and tools - Glioblastoma patient data including(MRI/CT scan data, as well as gene expression and anatomic characterization (ex. infiltrating vs cellular) of Glioblastoma tumor in situ hybridization (ISH) tissue sections and adjacent hematoxylin and eosin (H&amp;E)-stained sections. See <a href="http://ivygap.org/">the Ivy GAP database</a> and <a href="https://github.com/vjcitn/ivygapSE">ivygapSE</a> a tool that provides support for Summarized Experiment data - compatible for analysis in R for the tumor anitomical and gene expression data. See also <a href="https://itcr.cancer.gov/informatics-tools/ivyglimpse">ivyGlimpse</a> to explore tumor image data and how it relates to survival and gene expression. See <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6414061/">this article</a> and <a href="https://aapm.onlinelibrary.wiley.com/doi/abs/10.1002/mp.14556">this article</a> for more information.</li>
</ol>
<blockquote>
<p>“…an anatomically-based transcriptional atlas of human glioblastoma that aligns individual histologic features with genomic and gene expression patterns, thus assigning a molecular significance to the most important morphologic hallmarks of glioblastoma.”</p>
</blockquote>
<blockquote>
<p>To create the atlas, we surveyed the anatomic features by in situ hybridization (ISH), analyzed their transcriptomes by laser microdissection (LMD) and RNA sequencing (RNA-Seq), and validated the feature specific, gene expression enrichment of newly-identified markers by ISH (Fig. 1). We created a clinical and genomic database (<a href="http://ivygap.org/" class="uri">http://ivygap.org/</a>) for the 41-patient cohort (table S1) whose tumors (n=42) were evaluated to create the atlas. We describe gene sets whose expression is enriched in the anatomic features, measurements of intra- and inter-tumor heterogeneity, and a molecular subtype classification of transcriptomic samples from our atlas and The Cancer Genome Atlas (TCGA). Together, these two on line resources constitute the Ivy Glioblastoma Atlas Project (Ivy GAP).</p>
</blockquote>
<ol start="20" style="list-style-type: decimal">
<li>Cancer Imaging Phenomics Toolkit (CaPTk) - GUI or command line. web portal with tools for imaging and phenomics analysis for clinical decisions - currently focused on brain, breast, and lung cancer. In addition to image analysis tools for registration, segmentation, and feature extraction, it also aid daignostics with prognostic modeling, risk etimation and radiogenomics. See <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7402244/">this article</a> for more information. Designed for research purposes only and has not been reviewed or approved by the Food and Drug Administration or by any other agency. It is not intended or recommended for clinical applications.
<a href="https://youtu.be/WrjoJOxq4i8" class="uri">https://youtu.be/WrjoJOxq4i8</a></li>
</ol>
<p><img src="images/CaPTk.png" />
[<a href="https://cbica.github.io/CaPTk/index.html">Source</a>]</p>
<ol start="21" style="list-style-type: decimal">
<li>RADIOMICS.io - radiographic phenotyping tool using automated algorithms - allows for users to engineer features or to perform data driven analyses. See <a href="https://www.radiomics.io/science.html">here</a> for publications about using automated phenotyping using imaging.</li>
</ol>
<p><img src="images/radiomics.png" />
[<a href="https://www.nature.com/articles/ncomms5006">Source</a>]</p>
<p>Data transfer/sharing/management:
17) Globus is similar to Xsede seems like… as is codalab</p>
<p>Imaging analysis platforms:
18) cbibop - no updates since 2015??? think we should just show <a href="https://codalab.org/">codaLab Project</a> - open-source web-based platform that enables researchers, developers, and data scientists to collaborate, with the goal of advancing research fields where machine learning and advanced computation is used. <a href="https://youtu.be/WwFGfgf3-5s" class="uri">https://youtu.be/WwFGfgf3-5s</a>
experiment management system and easy collaboration system - includes competitions such as Liver tumor segmentation challenge</p>
<ol start="19" style="list-style-type: decimal">
<li><a href="https://www.xnat.org/about/">XNAT</a> - PET/CT (maybe MRI, MEG and EEG - look at connectomeDB???) data analysis</li>
</ol>
<ul>
<li>upload DICOM data , organize and share data, view, query, and download data, perform analysis through a secure API with access to high-powered computing clusters</li>
</ul>
<ol start="22" style="list-style-type: decimal">
<li>Region Templates…</li>
<li><a href="https://support.dcmtk.org/redmine/projects/dcmtk">DCMTK</a> - A set of command line tools for working with DICOM data, including examining and converting DICOM image files and data transfer and storage of images</li>
</ol>
</div>
<div id="galaxy" class="section level3">
<h3><span class="header-section-number">5.2.5</span> Galaxy</h3>
<p>This section was written by Jeremy Goecks:</p>
<p>Galaxy is a web-based computational workbench that connects analysis tools, biomedical datasets, computing resources, a graphical user interface, and a programmatic API (Figure 1). Galaxy (<a href="https://galaxyproject.org/" class="uri">https://galaxyproject.org/</a>) enables accessible, reproducible, and collaborative biomedical data science by anyone regardless of their informatics expertise. There are more than 8,000 analysis tools and 200 visualizations integrated into Galaxy that can be used to process a wide variety of biomedical datasets. This includes tools for analyzing genomic, transcriptomic (RNA-seq), proteomic, metabolomic, microbiome, and imaging datasets, tool suites for single-cell omics and machine learning, and thousands of more tools. Galaxy’s graphical user interface can be used with only a web browser, and there is a programmatic API for performing scripted and automated analyses with Galaxy.</p>
<p>Galaxy is used daily by thousands of scientists across the world. A vibrant Galaxy community has deployed hundreds of Galaxy servers across the world, including more than 150 public and three large national/international servers in the United States, Europe, and Australia (<a href="https://usegalaxy.org" class="uri">https://usegalaxy.org</a>, <a href="https://usegalaxy.eu" class="uri">https://usegalaxy.eu</a>, <a href="https://usegalaxy.org.au" class="uri">https://usegalaxy.org.au</a>). The three national/international servers (Figure 2) have more than 250,000 registered users who execute &gt;500,000 analysis jobs each month. Galaxy has been cited more than 10,000 times with &gt;20% from papers related to cancer. The Galaxy Tool Shed (<a href="https://usegalaxy.org/toolshed" class="uri">https://usegalaxy.org/toolshed</a>) provides a central location where developers can upload tools and visualizations and users can search and install tools and visualizations into any Galaxy server. Galaxy has a large presence in the cancer research community. Galaxy serves as an integration and/or analysis platform for 7 projects in the NCI ITCR program. There is also increasing use of Galaxy in key NIH initiatives such as the NCI Cancer Moonshot Human Tumor Atlas Network (HTAN) and the NHGRI Data Commons, called the AnVIL (<a href="https://anvilproject.org/" class="uri">https://anvilproject.org/</a>).</p>
<p>Galaxy’s user interface, accessible via a web browser, provides access to all Galaxy functionality. The main Galaxy interface (Figure 3) has three panels: available tools (left), running analyses and viewing data (middle), and a full history of tools run and datasets generated (right). Datasets for analysis in Galaxy can be uploaded from a laptop or desktop computer or obtained from public data repositories connected to Galaxy. With Galaxy, complex workflows composed of tens or even hundreds of analysis tools can be created and run. In Galaxy’s workflow interface (Figure 4), tools can be added and connected via a simple drag-and-drop approach. Galaxy users can share all their work—analysis histories, workflows, and visualizations—via simple URLs that are available to specific colleagues or a link that anyone can access. Galaxy’s user interface is highly scalable. Tens, hundreds, or even thousands of datasets can be grouped into collections and run in parallel using individual tools or multi-tool workflows. In summary, Galaxy is a popular computational workbench with tools and features for a wide variety of data analyses, and it has broad usage in cancer data analysis.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="itcr-tools.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="computing-for-cancer-research.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
